CRM/Upgrade/Incremental/sql/4.4.6.mysql.tpl

```diff
@@ -1,4 +0,0 @@
-{* file to handle db changes in 4.4.6 during upgrade *}
--- CRM-14903
-ALTER TABLE `civicrm_mapping_field`
-CHANGE COLUMN `operator` `operator` ENUM('=','!=','>','<','>=','<=','IN','NOT IN','LIKE','NOT LIKE','IS NULL','IS NOT NULL', 'IS EMPTY', 'IS NOT EMPTY', 'RLIKE');
```

Comment:
It never ran (I am deleting files that were skipped for clarity).
And it is immediately obsolete during the upgrade anyway -- the 4.5.alpha1 alters that column to be something else.
We could consider backporting it for the 4.4 LTS version, but for 4.6+ it can just be removed.